crowdsourcing/viewsets/task.py

```diff
@@ -261,3 +284,37 @@ class CurrencyViewSet(viewsets.ModelViewSet):
 
     queryset = Currency.objects.all()
     serializer_class = CurrencySerializer
+
+
+class ExternalSubmit(APIView):
+    def post(self, request, *args, **kwargs):
+        identifier = request.GET.get('daemo_id', False)
```

Comment:
Can you explain the user workflow if they're using AJAX? eg making the
submission in the background?

On Saturday, December 19, 2015, Durim Morina <notifications@github.com>
wrote:

> In crowdsourcing/viewsets/task.py
> <https://github.com/crowdresearch/crowdsource-platform/pull/604#discussion_r48095890>
> :
>
> > @@ -261,3 +284,37 @@ class CurrencyViewSet(viewsets.ModelViewSet):
> >
> >      queryset = Currency.objects.all()
> >      serializer_class = CurrencySerializer
> > +
> > +
> > +class ExternalSubmit(APIView):
> > +    def post(self, request, *args, **kwargs):
> > +        identifier = request.GET.get('daemo_id', False)
>
> use query_params
> and let's allow ajax calls as well request.POST wont be filled in that
> case use request.data
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crowdresearch/crowdsource-platform/pull/604/files#r48095890>
> .
>


-- 
sent from my NLS